src/App.test.tsx

```diff
@@ -0,0 +1,5 @@
+test("jest is working", ()=>{
+  expect(true).toBe(true);
+})
+
+export {}
```

Comment:
nit: no need to export on tests

Reply:
You are correct - no need to export. Added this to clean up a vscode error in my syntax highlighting, still not quite sure how to fix that. But it works regardless.

Reply:
fixed in commit 830e99a70897664a3d923b2d0f42c8e6053234cc